terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfView.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2013-2018 NTT DATA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package org.terasoluna.tourreservation.app.managereservation;
+
+import java.text.SimpleDateFormat;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.view.document.AbstractPdfStamperView;
+
+import com.lowagie.text.pdf.AcroFields;
+
+@Component
+public class ReservationReportPdfView extends AbstractPdfStamperView {
+
+    @Value("${reservation.report.pdf.path}")
+    String reservationReportPdfPath;
+
+    @Override
+    protected void mergePdfDocument(Map<String, Object> model,
+            com.lowagie.text.pdf.PdfStamper stamper, HttpServletRequest request,
+            HttpServletResponse response) throws Exception {
+        DownloadPDFOutput downloadPDFOutput = (DownloadPDFOutput) model.get(
+                "downloadPDFOutput");
+
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy/MM/dd");
+
+        AcroFields form = stamper.getAcroFields();
+        String referenceName = downloadPDFOutput.getReferenceName();
+        if (referenceName.length() >= 30) {
+            form.setFieldProperty("referenceName", "textsize", new Float(8),
+                    null);
+        }
```

Comment:
- [ ] Always text size 8 is not good?

Reply:
I wanted to decide the size based on the length of the character string since it caused a sense of discomfort in the display when a short character string came in if I always set the size of the character small. Is there something to worry about?

Reply:
If you really want to size due to data size, you must due to not character length but byte or codepoint length.

Reply:
7eb3b5d688ca01c8f12e917c5b4bfbcb48611af4

I changed to get data size by byte size in above commit .

Reply:
* You should erase Magic Number `30`, `8` and make it easy to change.
* Why use wrapper `Float`,  `8.0F` is not good?
* You should use `StandardCharsets.UTF_8`. (character encoding should be 
 `StandardCharsets.UTF_8.name()`).

Reply:
d320b0c8bf0fd56e1e9682610ddf7bbd95006103

I modified in above commit .

Reply:
I think that the `30`, `8` are enough to be as constant and no need to be as a property value.
Because when remake PDF template, you should re-consider size about not only `referenceName`.

Reply:
aec1d06174c75cc5eff01604bfd1d104cffb0a94

I fixed in above commit .